docs/bsp.md

```diff
@@ -1194,7 +1194,7 @@ trait ScalaMainClassesItem {
   def target: BuildTargetIdentifier
 
   /** The main class item. */
-  def mainClass: ScalaMainClass
+  def classes: List[ScalaMainClass]
```

Comment:
Why this change? A `ScalaMainClassItem` links a class to a build target identifier, and the result returns a list of `ScalaMainClassItem`s.

Reply:
This is synchronizing the spec with the current bsp4s implementation 
https://github.com/scalacenter/bsp/blob/49d840991f5e57ef3af1e535328c2baa179c1d75/src/main/scala/ch/epfl/scala/bsp/Bsp.scala#L543